src/services/link/index.js

```diff
@@ -113,15 +113,12 @@ module.exports = function setup() {
 				const expertSchoolId = data.esid; const { email } = data; const
 					{ teamId } = data;
 
-				const hashService = app.service('hash');
-				const linkService = app.service('link');
-
 				if (email) {
 					// generate import hash
 					const user = (await app.service('users').find({ query: { email: data.toHash } }) || {}).data[0];
 					if (user && user.importHash) linkInfo.hash = user.importHash;
 					else {
-						await hashService.create({
+						await app.service('hash').create({
```

Comment:
please cleanup this if/else and instead of then(generatedHash)... take the return value and assign it to linkInfo.hash, remote everything in then and catch, 
```suggestion
						linkInfo.hash = await app.service('hash').create({params})
```

Reply:
:+1:  done